web_listview_invert_selection/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2017 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'List Invert Selection',
+    'summary': """
+        Allows to invert the current selection of a list of records
```

Comment:
Summary could be shorter, eg.:
`    'summary': 'Invert current selection of list of records',`